src/components/MAPI/permissions/PermissionsOverview/PermissionsOverviewOrganizations.tsx

```diff
@@ -0,0 +1,48 @@
+import { selectOrganizations } from 'model/stores/organization/selectors';
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import PermissionsUseCases from '../../../UI/Display/MAPI/permissions/PermissionsUseCases';
+import { IPermissionsUseCase, PermissionsUseCaseStatuses } from '../types';
+import { usePermissions } from '../usePermissions';
+import { getStatusesForUseCases } from '../utils';
+
+interface IPermissionsOverviewOrganizationsProps {
+  useCases: IPermissionsUseCase[];
+}
+
+const PermissionsOverviewOrganizations: React.FC<
+  IPermissionsOverviewOrganizationsProps
+> = ({ useCases }) => {
+  const provider = window.config.info.general.provider;
+  const { data: permissions } = usePermissions();
+  const organizations = useSelector(selectOrganizations());
+  const sortedOrganizations = Object.values(organizations).sort((a, b) =>
+    (a?.name || a.id).localeCompare(b?.name || b.id)
+  );
+
+  const useCasesStatuses: PermissionsUseCaseStatuses = useMemo(() => {
+    if (typeof permissions === 'undefined') return {};
+
```

Comment:
Since we display the `<UseCasesPlaceholder />` when `useCaseStatuses` is falsey, could we take advantage of that here and return `undefined` (or another falsey value)? Then I think the check on line 35 to return `null` wouldn't be necessary anymore

Reply:
fixed